Matveibos/hometask-6/my_hash_spec.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require_relative './my_hash'
+require 'pry'
+require 'rspec'
+
+describe MyHash do
+  subject(:my_hash) { described_class.new }
+
+  describe '#[]=' do
+    context 'when value does not exist' do
+      it 'return pair key-value' do
+        expect(my_hash[:key1] = '').to eq('')
+        expect(my_hash[:key1]).to eq('')
+      end
+    end
+
+    context 'when value exist' do
+      before do
+        subject[:key1] = 54
+      end
+
+      it 'replace value' do
+        expect(my_hash[:key1]).to eq(54)
+        expect(my_hash[:key1] = 98).to eq(98)
+        expect(my_hash[:key1]).to eq(98)
```

Comment:
https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/change-matcher
Use this matcher for the test